CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 3.0.0-alpha.10
+
+-   upgrade the runtime to version 2.0.0-alpha.26
+-   routes for generating QrCodes for Files and RelationshipTemplates without Tokens
+
+## 3.0.0-alpha.9
+
+-   npm workspaces
```

Comment:
Ich dachte, in den changelog kommen nur Sachen rein, die der Nutzer sehen kann?

Reply:
ja, kp. fands komisch dass in der version nix steht. Wir haben ja die version nur erhöht um zu schauen dass beim publishen alles klappt.. aber ich kanns raus nehmen

Reply:
Mir isses egal. Der Changelog ist dein Baby. :-P